woocommerce/woocommerce
# Remove defunct AJAX event names.

### All Submissions:






### Changes proposed in this Pull Request:



Found while reviewing https://github.com/woocommerce/woocommerce/pull/30839.

The `increase_order_item_stock` and `reduce_order_item_stock` action callbacks were removed in https://github.com/woocommerce/woocommerce/commit/8261cffaa7aa69846f7f32d25e73117bae2ad25c#diff-fe11af15981497a5dd45f2519f4ce2c3daeff6646e6a10814c87b086ee7cd55a and the `add_order_item_meta` and `remove_order_item_meta` callbacks were removed in https://github.com/woocommerce/woocommerce/commit/21ad196297612b9ac94c1c871c0bd043dbed4004#diff-fe11af15981497a5dd45f2519f4ce2c3daeff6646e6a10814c87b086ee7cd55a. This PR cleans up the defunct action names.

### How to test the changes in this Pull Request:

1. Verify the functions of the same names do not exist in the `WC_AJAX` class
2. Optionally, use `wp.ajax.post()` in a developer console (on a wp-admin page) to verify no PHP Warning is triggered for `woocommerce_increase_order_item_stock`, etc)

### Other information:




### Changelog entry

> Dev - Remove defunct AJAX events.

### FOR PR REVIEWER ONLY:
